prow/prstatus/prstatus.go

```diff
@@ -26,6 +26,7 @@ import (
 	"time"
 
 	gogithub "github.com/google/go-github/github"
+	"github.com/gorilla/sessions"
```

Comment:
I don't feel good about adding an import just for the sake of the fix but I really wanted to extract session invalidation to a method (it's now used in two places) and for that I needed `sessions.Session` type. Maybe there's some better solution for that.

Reply:
What good is a package if it doesn't import every other package? At least this library is already vendored.